src/controllers/__tests__/ioSignController.test.ts

```diff
@@ -239,3 +319,193 @@ describe("IoSignController#getQtspClausesMetadata", () => {
     });
   });
 });
+
+describe("IoSignController#getSignatureRequest", () => {
```

Comment:
Shall we add some tests in case of some bad path? for example in case of Validation errors?